### ISSUE TITLE
UnknownStructureError 1.42 | Add support for the new scenarios from Update 50292  

### ISSUE THREAD

**Issue by LoboFJP** [KSneijders/AoE2ScenarioParser#20]
Hello. It is me again.
This time I think it has to do with the new version of AOE2, it had worked quite well in the last few weeks, however just today I get this error:

AoE2ScenarioParser.helper.exceptions.UnknownStructureError: The structure conditions could not be found with: DE:1.42

This is the code:

```
from AoE2ScenarioParser.scenarios.aoe2_de_scenario import AoE2DEScenario
from AoE2ScenarioParser.datasets.players import PlayerId
from AoE2ScenarioParser.objects.support.trigger_select import TS

input_path = r"C:\Users\xaero\Games\Age of Empires 2 DE\76561198062020840\resources\_common\scenario\CBA DoGo Edition.aoe2scenario"
output_path = r"C:\Users\xaero\Games\Age of Empires 2 DE\76561198062020840\resources\_common\scenario\CBA DoGo Edition.aoe2scenario"
scenario = AoE2DEScenario.from_file(input_path)

trigger_manager = scenario.trigger_manager
print(trigger_manager.get_summary_as_string())
```

T_T

**Comment by KSneijders** [comment]
Hey!
This happens because of the newest AoE2 patch that dropped a couple hours ago. They changed stuff about scenarios, so I have to update the parser. I hope to have time to figure out what they changed tonight. I'll keep you updated! 🙂

**Comment by KSneijders** [comment]
I just published version `0.1.10`. This new version supports the new 1.42 format. 
It also added 2 new datasets and a new effect they introduced without mentioning it in the patch notes 11. 

You can find the update command [here](https://aoe2scenarioparser.readthedocs.io/en/master/install.html#updating).

If there are any more issues, feel free to make a new issue!
